dodo.py

```diff
@@ -436,18 +431,30 @@ def task_dev():
             "--no-index",
             "--find-links",
             B.DIST,
-            C.NAME,
         ]
+        file_dep = []
+        for py_name in [C.NAME, C.CORE_NAME]:
+            py_name_pkg = py_name.replace("-", "_")
+            file_dep += [B.DIST / f"""{py_name_pkg}-{D.PY_VERSION}-{C.NOARCH_WHL}"""]
+            args += [py_name]
+
+        yield dict(
+            name=f"py:jupyterlite-core",
+            actions=[U.do(*args, cwd=P.ROOT)],
+            file_dep=file_dep,
+        )
     else:
-        cwd = P.PY_SETUP_PY[C.NAME].parent
-        file_dep = [cwd / "src" / C.NAME / B.APP_PACK.name]
-        args = [*C.FLIT, "install", "--pth-file", "--deps=none"]
+        for py_name in [C.NAME, C.CORE_NAME]:
+            py_name_pkg = py_name.replace("-", "_")
+            cwd = P.PY_SETUP_PY[py_name].parent
+            file_dep = [cwd / "src" / py_name_pkg / B.APP_PACK.name]
+            args = [*C.FLIT, "install", "--pth-file", "--deps=none"]
```

Comment:
I don't recall why we have two ways of installing the Python packages depending on whether it's running locally or on CI.

Reply:
We don't need to fix this in this PR but at some point it would be nice to enable more familiar for installing the packages like `python -m pip install -e .`.

Reply:
> two ways of installing the Python packages depending on whether it's running locally or on CI.

In CI, testing against as-built assets is a general way to avoid "surprises," which get amplified with more complex, multi-language packages and, of course, various build backends.

For a while `pip` broke editable installs by anything other than `setuptools-meta` (and somehow blamed the backends), so I haven't been too excited to switch back.

Reply:
Sure. I mean we already have a dist task and this is for dev.

Anyway not gonna look into this now.